src/BackupCode/VerifyHandler.php

```diff
@@ -67,7 +67,7 @@ public function verify(HTTPRequest $request, StoreInterface $store, RegisteredMe
             );
         }
 
-        $code = $bodyJSON['code'];
+        $code = preg_replace("/[^A-Za-z0-9]/", '', $bodyJSON['code']);
```

Comment:
Are these definitely the only valid characters? Is there some documentation or such that you can link to about that?

Reply:
It might be safer to _just_ strip out the spaces, since the issue this aims to fix is specifically that copying the code from the UI results in spaces that should not be there.

Reply:
Thanks Guy, I have updated that